blocks/search/search.css

```diff
@@ -0,0 +1,276 @@
+/* search box */
+.search .search-box {
```

Comment:
all the `.search` references should be `.search.block` or `.block.search`

Reply:
@kronnox done!